packages/search-box/README.zh-CN.md

```
@@ -0,0 +1,135 @@
+- 强大的搜索功能，支持模糊搜索、自定义搜索等
+
+![TinySearchBox](TinySearchBox.png)
+
+## 快速上手
+
+安装 TinySearchBox
+
+```shell
+npm i @opentiny/vue-search-box
+```
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)